Formula/a/autobrr.rb

```diff
@@ -0,0 +1,52 @@
+class Autobrr < Formula
+  desc "Modern, easy to use download automation for torrents and usenet"
+  homepage "https://github.com/autobrr/autobrr"
+  url "https://github.com/autobrr/autobrr/archive/refs/tags/v1.36.0.tar.gz"
+  sha256 "631ee5f858c5952e1d7a8c27b13fe33aea9021ffd814bb540f0e494291168309"
+  license "GPL-2.0-or-later"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    # setting up Go environment variables
+    ENV["GO111MODULE"] = "on"
+    ENV["GOPROXY"] = "https://proxy.golang.org,direct"
+
+    # build the frontend
+    system "npx", "pnpm", "install", "--dir", "web"
+    system "npx", "pnpm", "--dir", "web", "run", "build"
+
+    # compile flags to embed version information
+    ldflags = "-s -w -X main.version=#{version} -X main.commit=HOMEBREW"
```

Comment:
`#{tap.user}` == `Homebrew`

```suggestion
    ldflags = "-s -w -X main.version=#{version} -X main.commit=#{tap.user}"
```

Reply:
I'm getting the following error with this method:

```console
Error: An exception occurred within a child process:
  NoMethodError: undefined method `user' for nil:NilClass
```
I'm running brew on a MacBook M1. I have still committed your suggestion though 👍